CheckYourEligibility-Parent/wwwroot/js/cookieFunctions.js

```diff
@@ -20,28 +20,55 @@ function initializeClarity() {
 }
 
 function initCookieConsent() {
+    console.log("in innitCookieConsent");
     const hasChoice = cookie.read("cookie");
-    if (hasChoice==="true") {
+    if (hasChoice !== null) {
         initializeClarity();
-    }
-    
-    else if (hasChoice !== "false") {
+        document.getElementById('cookie-banner').style.display = 'none';
+        console.log("in innitCookieConsent - hasChoice not equal null");
+
+    } else {
         document.getElementById('cookie-banner').style.display = 'block';
+        console.log("in innitCookieConsent - hasChoice else");
     }
-    
-    console.log(hasChoice);
 }
 
-document.getElementById('accept-cookies').onclick = function() {
+document.getElementById('accept-cookies').onclick = function () {
     cookie.create("cookie", "true", 365);
     document.getElementById('cookie-banner').style.display = 'none';
     initializeClarity();
 };
-document.getElementById('reject-cookies').onclick = function() {
+document.getElementById('reject-cookies').onclick = function () {
     cookie.create("cookie", "false", 365);
     document.getElementById('cookie-banner').style.display = 'none';
 };
 
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
This wrapper isn't needed, because page is already loaded by time JS is executed.